system/lib/pthread/proxying.c

```diff
@@ -35,6 +35,8 @@ static em_proxying_queue system_proxying_queue = {
   .capacity = 0,
 };
 
+static _Thread_local int system_queue_in_use = 0;
```

Comment:
We could use `bool` here along with `true/false` but maybe thats a bigger/unnecessary change.

Reply:
This was switch to bool in #24596, so this can be updated now I think.